.travis/deploy-master.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+eval "$(ssh-agent -s)" # Start ssh-agent cache
+chmod 600 .travis/id_rsa # Allow read access to the private key
+ssh-add .travis/id_rsa # Add the private key to SSH
+
+cd ..
+git clone git@github.com/tulibraries/ansible-playbook-airflow.git # clone deployment playbook
+cd ansible-playbook-airflow
+sudo pip install pipenv # install playbook requirements
+pipenv install # install playbook requirements
+pipenv run ansible-galaxy -r requirements.yml # install playbook role requirements
+cp .circleci/.vault ~/.vault # setup vault password retrieval from travis envvar
```

Comment:
is this circle ci specific or also available on travis?

Reply:
no - its looking in the .circleci folder on the playbook just bc it was originally put there for circle deployments, but the script itself is just a bash script that checks for an envvar with the vault password